pr-deploy/package.json

```diff
@@ -13,6 +13,7 @@
     "node": "10.16.0"
   },
   "scripts": {
+    "build": "node ~/src/TypeScript/built/local/tsc.js ./src/*.ts --esModuleInterop",
```

Comment:
whats `esModuleInterop` for? (just for my knowledge)

Reply:
The type defs for some of our dependencies declare modules using `export = modulename` which can only be default-imported using this flag. I don't understand it 100% but here's the docs - https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
Also I had to manually compile `tsc.js` so a future step is to write a build script that compiles it for you